app/cafe.py

```
@@ -0,0 +1,27 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def __str__(self) -> str:
+        return self.name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if (
+                visitor.get("vaccine") is None
+                or visitor["vaccine"].get("expiration_date") is None
+        ):
+            raise NotVaccinatedError
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if datetime.date.today() > expiration_date:
+            raise OutdatedVaccineError
+        if visitor["wearing_a_mask"] is False:
+            raise NotWearingMaskError(0)
```

The `NotWearingMaskError` is raised with a hardcoded value `0`. According to the checklist, custom exceptions should be raised with the required arguments. Please ensure you pass the correct visitor information, such as the visitor's name, to the exception.